controllers/cluster_controller.go

```diff
@@ -64,6 +64,7 @@ const (
 // ClusterReconciler reconciles a Cluster object
 type ClusterReconciler struct {
 	Client client.Client
+	Config
```

Comment:
Since the config only has one field, it might be preferable to pass `WatchLabelValue` into Reconcilers as a field directly (i.e. `Config` -> `WatchLabelValue string`).

Reply:
I was going back and forth on this. I agree that it would currently make more sense to just pass the string along.

Reply:
I think I will bring this up when I make the upstream PR - let them decide which option they prefer